### PR TITLE
Feat/semi theme

### PR DESCRIPTION
**Pull request by zamhown**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vchart-theme/src/semi-design/common/constants.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const SEMI_FONT_FAMILY =
```
Comment (xile611) [inline packages/vchart-theme/src/semi-design/common/constants.ts:1]: semi-design 主题可以单独抽一个主题包吗？

File: packages/vchart-theme/src/v-screen/index.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { getVScreenCommonTheme } from './common';
 export const vScreenThemeList = Object.keys(vScreenColorScheme).map(name =>
   getVScreenCommonTheme(
     `vScreen${name[0].toUpperCase()}${name.slice(1)}`,
-    vScreenColorScheme[name].name,
+    `大屏-${vScreenColorScheme[name].name}`,
```
Comment (xile611) [inline packages/vchart-theme/src/v-screen/index.ts:7]: 为什么有中文的key??

File: packages/vchart-theme/src/semi-design/common/constants.ts (hunk 1 above)
Comment (zamhown) [reply packages/vchart-theme/src/semi-design/common/constants.ts:1]: @xile611 可以的，这个包要和 vchart-theme 一样保持和 vchart 互不依赖吗

File: packages/vchart-theme/src/v-screen/index.ts (hunk 2 above)
Comment (zamhown) [reply packages/vchart-theme/src/v-screen/index.ts:7]: > 为什么有中文的key??

这个是主题的 description，后面的 `vScreenColorScheme[name].name` 是大屏那边的原始中文名称，目前 description 只用在了 readme 的自动生成（https://github.com/VisActor/VChart/blob/feat/semi-theme/packages/vchart-theme/README.md）。

去掉也可以？不过会丢失大屏那边原始的的中文信息

File: packages/vchart-theme/src/semi-design/common/constants.ts (hunk 1 above)
Comment (xile611) [reply packages/vchart-theme/src/semi-design/common/constants.ts:1]: 比如 vchart-semi-theme，应该在 peerDependencies 中声明vchart版本吧

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**